spec/dealership_spec.rb

```diff
@@ -0,0 +1,14 @@
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+  it 'exists' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.inventory). to eq([])
+  end
```

Comment:
Since you are tesing that "it exists", I would only test that the dealership is a correct type. Then I would test if it had an inventory separately:
```suggestion
  it 'exists' do
    dealership = Dealership.new("Acme Auto", "123 Main Street")
    expect(dealership).to be_a Dealership
  end
  
   it 'has an inventory' do
    dealership = Dealership.new("Acme Auto", "123 Main Street")
    expect(dealership.inventory).to eq([])
  end
```